torchrl/objectives/dreamer.py

```diff
@@ -293,6 +325,10 @@ def forward(self, tensordict: TensorDict) -> Tuple[TensorDict, TensorDict]:
         else:
             actor_loss = -lambda_target.sum((-2, -1)).mean()
         loss_tensordict = TensorDict({"loss_actor": actor_loss}, [])
+        if self.return_tensorclass:
+            return DreamerModelLosses._from_tensordict(
+                loss_tensordict
+            ), DreamerModelLosses._from_tensordict(fake_data.detach())
```

Comment:
Can you explain this? Why do we make a second tensorclass? Shouldn't it still be a tensordict?